lib/internal/policy/manifest.js

```diff
@@ -499,8 +500,7 @@ class Manifest {
             timingSafeEqual(digest, expected)) {
             return true;
           }
-          MapPrototypeSet(
-            realIntegrities,
+          realIntegrities.set(
```

Comment:
This leaks into user code via the `throw`, we don't want to expose SafeMap to user code was last I remember

Reply:
I don't think it leaks to user code:

https://github.com/nodejs/node/blob/219332e66cd5dec76f77eefb9e1e9698ef246ba6/lib/internal/errors.js#L1226-L1241

The SafeMap values are passed as strings to the error message.